java/src/main/native/src/CuFileJni.cpp

```diff
@@ -20,6 +20,8 @@
 #include <unistd.h>
```

Comment:
2021 copyrights

Reply:
Done.

Reply:
Older years should not be dropped.  This needs to be `2020-2021`.